squire_web/src/tournament/pairings/mod.rs

```diff
@@ -43,27 +45,32 @@ impl ActiveRoundSummary {
 }
 
 #[derive(Debug, PartialEq, Properties, Clone)]
-pub struct PairingsViewProps {
-    pub id: TournamentId,
-    pub admin_id: AdminId,
-    pub send_op_result: Callback<OpResult>,
-}
+pub struct PairingsViewProps {}
 
 #[derive(Debug, PartialEq, Clone)]
 pub enum PairingsViewMessage {
-    PlayerNamesReady(HashMap<PlayerId, String>),
+    /* Interaction */
     ChangeMode(PairingsViewMode),
+    PopoutActiveRounds(),
+    /* Update tournament */
     GeneratePairings,
     PairingsToRounds,
-    PairingsReady(PairingsWrapper),
-    ActiveRoundsReady(Vec<ActiveRoundSummary>),
-    PopoutActiveRounds(),
-    MatchSizeReady(u8),
     CreateSingleRound(),
     CreateSingleBye(),
     SingleRoundInput(usize, String),
     SingleByeInput(String),
```

Comment:
nit: Remove the `()` on variants that don't need them
```suggestion
    /* Interaction */
    ChangeMode(PairingsViewMode),
    PopoutActiveRounds,
    /* Update tournament */
    GeneratePairings,
    PairingsToRounds,
    CreateSingleRound,
    CreateSingleBye,
    SingleRoundInput(usize, String),
    SingleByeInput(String),
```